clang/lib/Analysis/UnsafeBufferUsage.cpp

```diff
@@ -721,6 +721,33 @@ class UnsafeBufferUsageAttrGadget : public WarningGadget {
   DeclUseList getClaimedVarUseSites() const override { return {}; }
 };
 
+// Warning gadget for unsafe invocation of span::data method.
+// Triggers when the pointer returned by the invocation is immediately
+// cast to a larger type.
+
+class DataInvocationGadget : public WarningGadget {
+  constexpr static const char *const OpTag = "data_invocation_expr";
```

Comment:
a nitpick:  I noticed most tag strings use camel cases.   We better be consistent on this?

Reply:
Thank you. Will fix this.